library/src/test/java/io/ona/kujaku/layers/ArrowLineLayerTest.java

```diff
@@ -720,4 +721,21 @@ public void filterFeaturesByRegex() throws InvalidArrowLineConfigException {
         assertEquals(4, filteredFeatures.get(2).getNumberProperty("position"));
         assertEquals(5, filteredFeatures.get(3).getNumberProperty("position"));
     }
+
+    @Test
+    public void getLayerIds() throws InvalidArrowLineConfigException {
```

Comment:
Could you maybe use the convention where you start with the suffix `test` and say what to expect from the test (in the function name)?

Reply:
Cool, there are many conventions on naming tests? Should we mix the conventions in one repository? The mostly used convention is **methodWhenGivenShouldDo** or **methodShouldDoWhenGiven** applied appropriately depending on the test. In this case, I probably forgot to include the implicit return expected

Reply:
`testMethodShouldDoWhenGiven` works.

Reply:
Please counter-check this.

I think you left out the `ShouldDoWhenGiven`. I guess the point would be to follow one convention across all Kujaku tests?

Reply:
Yea, I decided to use `testGetLayerIds` as you requested initially instead of mixing conventions. a test named `methodShouldDoWhenGiven` feels more easily readable compared to `testMethodShouldDoWhenGiven` and since this method has only a single expected outcome or rather no branching. It has one test for which it just returns (like a getter). At this point it would make sense not to name it as the method and have the `test` prefix for easy readability. Are you OK with this?

Reply:
Please choose one convention and carry it through whether or not there is branching in the method or not.

You can add a comment stating whatever convention you choose to use.